.github/workflows/release.yml

```diff
@@ -0,0 +1,78 @@
+name: Build and Push Docker Image
+
+on:
+  push:
+    branches:
+      - main
+    tags:        
+      - 'aci/helloworld:v*'
```

Comment:
should these be doing something like semver? like 1.0.0 instead of v1

Reply:
@BryceDFisher mentioned mimic hcsshim's versioning, so it will start with v and if it has a -rc, it's a pre-release build

Reply:
v1.0.0 is a production release.  v1.0.0-rc1 is a pre-release candidate for v1.0.0

Reply:
gotcha, resolving